src/utils/beacon/decodePayload.ts

```diff
@@ -0,0 +1,52 @@
+import { hex2buf } from "@taquito/utils";
+
+const PAYLOAD_PADDING = "0501";
```

Comment:
Should it be `PAYLOAD_PREFIX` or `OPTIONAL_PAYLOAD_PREFIX` considering there might be zeros as padding as well?

Reply:
A comment in code for what this prefix means/does would also be good 🙂